### DISCUSSION TITLE
[Build is Failing] on pushing code.

### DISCUSSION THREAD

**Discussion by ThisIsFaar** [joomla/joomla-cms#34560] [Q&A]
![image](https://user-images.githubusercontent.com/67395591/122603644-48327100-d092-11eb-9079-6b185033cd4d.png)
What is the reason behind this "Failure: Invalid Credentials" can anyone explain.

**Comment by richard67** [comment]
The issue is known, and people are already working on it.

The analysis4x uses RIPS, and this requires a login which seems to have expired.

**Reply by ThisIsFaar** [reply]
So what I have to do now, is there is any quick or temporary fix that can bypass this error

**Reply by richard67** [reply]
> So what I have to do now, is there is any quick or temporary fix that can bypass this error

No. But what's your problem? That test runs at the very end of the checks, so it does not block other checks to run, and you always can check results of drone to see which of the tests failed.

**Reply by ThisIsFaar** [reply]
My problem is I am not able to push the code in my pull request which needs some revision.
Yes, that test case runs at the end.
you can also check the last test case which is failing
![image](https://user-images.githubusercontent.com/67395591/122656734-1943fa00-d17b-11eb-8673-ae7337510394.png)


So, what is the solution for this?

**Reply by richard67** [reply]
The failing test should not keep you from pushing changes to your branch. If that does not work then somethlng else is wrong on your side, just check other PRs which have the same problem with the drone check, they can be opened, changes can be made. So I don’t understand what your problem is.
